src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
+            const options2 = [
+                { label: 'test1', value: '1' },
+                { label: 'test2', value: '2' },
+            ];
 
-        expect(getDefaultValueFromOptions(options)).toEqual('test1');
-        expect(getDefaultValueFromOptions(options2)).toEqual('1');
+            expect(getDefaultValueFromOptions(options2)).toEqual('1');
+        });
     });
+    describe('getFieldType', () => {
+        it('should get field type from field', () => {
+            const fooField = {
+                type: String,
+            };
 
-    it('should get field type from field', () => {
-        const fooField = {
-            type: String,
-        };
-        const barField = {
-            type: [String],
-        };
-        expect(getFieldType(fooField)).toBe(String);
-        expect(getFieldType(barField)).toBe(String);
-    });
+            expect(getFieldType(fooField)).toBe(String);
+        });
+        it('should get field type from field when type is array of type', () => {
+            const fooField = {
+                type: [String],
+            };
 
-    it('should return function name', () => {
-        const barFunction = function bar() {};
-        const fooFunction = function foo() {};
-        Object.defineProperty(fooFunction, 'name', { value: undefined });
-        expect(getFunctionName(barFunction)).toBe('bar');
-        expect(getFunctionName(String)).toBe('String');
-        expect(getFunctionName(Number)).toBe('Number');
-        expect(getFunctionName(Object)).toBe('Object');
-        expect(getFunctionName(Array)).toBe('Array');
-        expect(getFunctionName(Boolean)).toBe('Boolean');
-        expect(getFunctionName(Date)).toBe('Date');
-        expect(getFunctionName(fooFunction)).toBe('foo');
+            expect(getFieldType(fooField)).toBe(String);
+        });
+    });
+    describe('getFunctionName', () => {
+        it('should return function name of String', () => {
+            expect(getFunctionName(String)).toBe('String');
+        });
+        it('should return function name of Number', () => {
+            expect(getFunctionName(Number)).toBe('Number');
+        });
+        it('should return function name of Object', () => {
+            expect(getFunctionName(Object)).toBe('Object');
+        });
+        it('should return function name of Array', () => {
+            expect(getFunctionName(Array)).toBe('Array');
+        });
+        it('should return function name of Boolean', () => {
+            expect(getFunctionName(Boolean)).toBe('Boolean');
+        });
+        it('should return function name of Date', () => {
+            expect(getFunctionName(Date)).toBe('Date');
+        });
+        it('should return function name of fooFunction', () => {
+            const fooFunction = function foo() {};
+            Object.defineProperty(fooFunction, 'name', { value: undefined });
+            expect(getFunctionName(fooFunction)).toBe('foo');
+        });
```

Comment:
There is no case when `{ value: 'fooName' }`.

Reply:
added